engine/src/main/java/org/terasology/rendering/dag/RenderPipeline.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag;
+
+import java.util.ArrayList;
+
+/**
+ *
+ */
+public class RenderPipeline extends ArrayList<Object> {
```

Comment:
Oh, I see.

As mentioned below, I agree with removing this one. For debugging purposes, I don't have any idea for now. It would be clear when I need it, I think.